### ISSUE TITLE
Automatically compile `scss` stylesheets

### ISSUE THREAD

**Issue by hectorsector** [github/training-kit#497]
## Overview
**TL;DR**
Stylesheets are supposed to be automatically compiled from the `scss` source files, but they're not. This was [discussed with the last UI update](https://github.com/github/training-kit/pull/478#issuecomment-289380291).

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!
